### ISSUE TITLE
`set -o pipefail` breaks creduce

### ISSUE THREAD

**Issue by Kojoley** [csmith-project/creduce#190]
If I use `set -o pipefail` in a test script, creduce does not pass a precheck and tells me to ensure that the script is working correctly, and it does.

Reproducer (with any input file):
```sh
set -o pipefail
echo hello
```

**Comment by regehr** [comment]
does this issue matter?

**Comment by Kojoley** [comment]
What do you mean? Does it breaks some use cases? Yes it does, otherwise I would not hit it.

**Comment by regehr** [comment]
I'm asking whether removing this option from your bash script is feasible

**Comment by Kojoley** [comment]
Without rewriting it - no. And the workaround is to redirect to temporary files and check return codes yourself.

It would be good to document this if it is not going to be fixed. The other problem that other people will also spent some time on this ensuring that they did nothing wrong, but it is a creduce problem, because the error message suggests that the script does not work.

**Comment by regehr** [comment]
I agree it would be good to fix, but I doubt I'll have time in the immediate future. Patches are welcome.
